README.md

```diff
@@ -1,18 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> Learning how to add linter files 
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+
+-CSS
+- HTML
 
 ## Live Demo
 
```

Comment:
- [ ] [OPTIONAL] It's good to use a Live demo so you can show people your work, always make sure on having a working link.

Reply:
I have done everything except this one